src/commands/lambda/prompt.ts

```diff
@@ -125,18 +121,31 @@ export const datadogEnvVarsQuestions = (datadogApiKeyType: Record<string, any>):
   },
 })
 
-export const confirmationQuestion = (message: string): ConfirmQuestion => ({
+export const confirmationQuestion = (message: string): inquirer.ConfirmQuestion => ({
   message,
   name: 'confirmation',
   type: 'confirm',
 })
 
-export const functionSelectionQuestion = (functionNames: string[]): CheckboxQuestion => ({
+export const functionSelectionQuestion = (functionNames: string[]): CheckboxPlusPrompt => ({
   choices: functionNames,
-  message: 'Select the functions to modify',
+  highlight: true,
+  message:
+    'Select the functions to modify (Press <space> to select, p.s. start typing the name instead of manually scrolling)',
   name: 'functions',
-  type: 'checkbox',
-  validate: (selectedFunctions) => {
+  pageSize: 10,
+  searchable: true,
+  source: (answersSoFar: any, input: any) => {
```

Comment:
Can we type those? If not, prefer `unknown`

Reply:
updated in latest commit